ui/apps/platform/src/Containers/VulnMgmt/Entity/RelatedEntitiesSideList.js

```diff
@@ -25,99 +26,67 @@ const RelatedEntitiesSideList = ({ entityType, data, altCountKeyMap, entityConte
     const countKeyMap = { ...defaultCountKeyMap, ...altCountKeyMap };
 
     const matches = getEntityTypesByRelationship(entityType, relationshipTypes.MATCHES, useCase)
+        // @TODO: Remove the following filter step once ROX_FRONTEND_VM_UPDATES is ON
+        .filter((match) => {
```

Comment:
nit: This is fine how it is, but to be consistent with the suggestion above, maybe `matchedEntity` makes more sense